File: Cliente em Python e Ruby/cliente.py
from suds.client import Client
url = "http://localhost:10000/conversor?wsdl"
client = Client(url)
escolT = int(input("Bem-vindo ao Conversor de Temperaturas, digite: \n1 para digitar em Celsius: \n2 para digitar em Fahrenheit:\n "))
temp = int(input("Digite o valor da temperatura: "))
response = client.service.conversor(escolT,temp)
print(response)

File: Cliente em Python e Ruby/cliente.rb
require 'savon'

client  = Savon.client(wsdl: 'http://localhost:10000/conversor?wsdl')

puts "Bem-vindo ao Conversor de Temperaturas, digite: \n1 para digitar em Celsius: \n2 para digitar em Fahrenheit: "
escolT = gets

puts "Digite o valor da temperatura: "
temp = gets

response = client.call(:conversor, message: { escolhaN: escolT, temperat: temp})

puts response.body[:conversor_response][:return]